frontend/src/container/MetricsApplication/Tabs/Overview.tsx

```diff
@@ -64,12 +83,59 @@ function Application(): JSX.Element {
 		[handleSetTimeStamp],
 	);
 
-	const { topOperations, serviceOverview, topLevelOperations } = useSelector<
-		AppState,
-		MetricReducer
-	>((state) => state.metrics);
+	const serviceOverviewQuery = {
+		queryKey: [servicename, selectedTags, minTime, maxTime],
+		queryFn: (): Promise<PayloadProps> =>
+			getServiceOverview({
+				service: servicename || '',
+				start: minTime,
+				end: maxTime,
+				step: getStep({
+					start: minTime,
+					end: maxTime,
+					inputFormat: 'ns',
+				}),
+				selectedTags,
+			}),
+	};
+
+	const topOperationsQuery = {
+		queryKey: [minTime, maxTime, servicename, selectedTags],
+		queryFn: (): Promise<PayloadPropsTopOpertions> =>
+			getTopOperations({
+				service: servicename || '',
+				start: minTime,
+				end: maxTime,
+				selectedTags,
+			}),
+	};
 
-	const { queries } = useResourceAttribute();
+	const topLevelOperationsQuery = {
+		queryKey: [servicename, minTime, maxTime, selectedTags],
+		queryFn: (): Promise<ServiceDataProps> => getTopLevelOperations(),
+	};
+
+	const queryResult = useQueries([
```

Comment:
use type here for query's as you don't need to enforce the type

Reply:
Okay